tsconfig.json

```diff
@@ -1,15 +1,22 @@
 {
     "compilerOptions": {
-        "outDir": "./build",
+        "target": "es5",
+        "lib": ["dom", "dom.iterable", "esnext"],
         "allowJs": true,
-        "target":"es5",
+        "skipLibCheck": true,
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true,
+        "strict": true,
+        "forceConsistentCasingInFileNames": true,
+        "module": "esnext",
+        "moduleResolution": "node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "noEmit": true,
+        "jsx": "react",
+        "outDir": "./build",
         "noImplicitAny": true,
         "strictNullChecks": true
     },
-    "include": [
-        "./src/**/*"
-    ],
-    "exclude": [
-        "./node_modules"
-    ]
+    "include": ["src", "backend/src/**/*"]
```

Comment:
running $tsc throws a lot of errors and warnings, from src. additionally, the ./build directory no longer seems to be created. even after removing "src' from include (no errors get thrown), ./build still doesn't get built.